src/deploy/functions/discovery/jsexports/parseTriggers.ts

```diff
@@ -179,6 +177,7 @@ export function addResourcesToBackend(
       entryPoint: annotation.entryPoint,
       runtime: runtime,
       trigger: trigger,
+      environmentVariables: envs,
```

Comment:
Hmm.. this gives me a bit of pause. If we're requiring project-level env, should env be a backend attribute and not a FunctionSpec attribute?

Reply:
Woo that's an excellent point! Thanks - will address.